src/methods/geom_relations/overlaps.jl

```diff
@@ -76,6 +84,14 @@ overlaps(geom1, geom2)::Bool = overlaps(
     geom2,
 )
 
+
+# # Convert features to geometries
+overlaps(::GI.FeatureTrait, g1, ::Any, g2) = overlaps(GI.geometry(g1), g2)
+overlaps(::Any, g1, t2::GI.FeatureTrait, g2) = overlaps(g1, GI.geometry(g2))
+overlaps(::FeatureTrait, g1, ::FeatureTrait, g2) = overlaps(GI.geometry(g1), GI.geometry(g2))
```

Comment:
This is the only thing that CI doesn't cover.  We could remove it - but TBH it should be fine I think.